### ISSUE TITLE
[Feature request]: comments pagination handling

### ISSUE THREAD

**Issue by 3l3ktr0n** [e107inc/e107#5233]
### Motivation

nextprev feels outdated

### Proposed Solution

by today's standards a 'load more' approach would be more convenient

### Alternatives

or remove the pagination for comments

**Comment by Vodhin** [comment]
I agree that a "Load More" function would be nicer. This is typically accomplished via AJAX ands JavaScript/jQuery, both of which e107 v2 now include however it seems that the e107 AJAX calls I've looked at load into a dreaded iFrame which is not fashionable these days (probably to run any e107's PHP security checks on the request?). The plugins that I work on include my own code to do this kind of behind the scenes interaction, and tend to get rather bloated when incorporating those e107 security checks and other work arounds. 

In short, it wouldn't be too difficult to implement a "Load More" function but I'm already 1 ¾  years into coding my Real Estate plugin and am on the verge of having it ready for beta release. I don't have any websites that use comments, but I could look into writing some JavaScript that could replace the Next/Previous buttons (would work nearly everywhere, the way I'm thinking of it).

**Comment by 3l3ktr0n** [comment]
has anyone looked into this?
